src/transport.ts

```diff
@@ -142,7 +141,7 @@ export class WebRTCTransport implements Transport {
       dataChannelOpenPromise.reject(dataChannelError('data', error))
     }
 
-    const ufrag = 'libp2p+webrtc+v1/' + genUuid().replaceAll('-', '')
+    const ufrag = 'libp2p+webrtc+v1/' + genUfrag(20)
```

Comment:
Is the new random identifier intentionally shorter (20 bytes) than the old one (32 bytes)?

Reply:
fixed. thanks for pointing this out.